.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ commands:
           name: Build
           command: |
             cd <<parameters.project>>
-            flutter packages pub run build_runner build
+            flutter packages pub run build_runner build --delete-conflicting-outputs
```

Comment:
Also made this change in https://github.com/RevenueCat/purchases-flutter/pull/285 to see if tests pass. Whichever PR gets merged first will include the change